apps/website/src/content/docs/Kbd.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Kbd
+description: This component is used to display which key or combination of keys performs a given action.
+thumbnail: Keyboard
+---
+
+<p>{frontmatter.description}</p>
+
+<LiveExample src='Kbd.main.tsx' truncate={false}>
+  <AllExamples.KbdMainExample client:load />
+</LiveExample>
+
+## Usage
+
+The `<Kbd>` component is a styled wrapper around the [`<kbd>` HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd).
+
+You can either pass custom key(s) as the `children` or use the `KbdKeys` enum which includes a convenient set of constants for common keyboard keys such as:
```

Comment:
Since you are listing out all values, "such as" will not be needed.

```suggestion
You can either pass custom key(s) as the `children` or use the `KbdKeys` enum which includes some of the common keyboard keys:
```